### PR TITLE
fix: update cosmic swingset build instructions

### PR DESCRIPTION
**Pull request by JimLarson**
Unfortunately, with no `make clean` target, there doesn't seem to be a way to test this short of manually cleaning `~/go/bin` and starting from a fresh git clone. However, this change is definitely an improvement on the instructions.

Consider whether we should to refer to https://agoric.com/documentation/getting-started/before-using-agoric.html instead of duplicating the contents.

### CODE REVIEW

File: packages/cosmic-swingset/README.md [hunk 1]
```diff
@@ -23,10 +23,9 @@ In any case, for now, you will be needing to build the solo node from the source
 
 If you want to build and install from sources, you need to install
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- [node.js](https://nodejs.org/en/) (you need at least version 11). This installs 2 binaries named `node` (JavaScript runtime) and `npm` (JavaScript package manager)
```
Comment (michaelfig) [inline packages/cosmic-swingset/README.md:26]: Yes, please remove as much of this as is duplicated by (and point to) the documentation site.

**Review by michaelfig — APPROVED** [review]
Better than before, so feel free to land.  If you have time, your suggestion of removing duplication with the doc site is a good one.

File: packages/cosmic-swingset/README.md (hunk 1 above)
Comment (JimLarson) [reply packages/cosmic-swingset/README.md:26]: Done.

**Review by michaelfig — APPROVED** [review]
LGTM!

**Pull request merged**